src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
```

Comment:
you can use destructuring here 
```suggestion
export const CommentInfo: React.FC<Props> = ({ comment }) => {
  const { name, email, body } = comment;

  return (

```